<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>examwindow</class>
 <widget class="QDialog" name="examwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 213, 197);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>791</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="examgrid">
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>50</number>
    </property>
    <item row="0" column="0" colspan="5">
     <widget class="QLabel" name="label_exam">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 25pt &quot;Copperplate Gothic Light&quot;;
	color: rgb(99, 45, 59);
}</string>
      </property>
      <property name="text">
       <string>Exam</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_examlist">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 25pt &quot;Copperplate Gothic Light&quot;;
	color: rgb(99, 45, 59);
}</string>
      </property>
      <property name="text">
       <string>Exam List</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_examname">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 16pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
}</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QDateTimeEdit" name="datetime_exam">
      <property name="styleSheet">
       <string notr="true">QDateTimeEdit{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
}</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_examdes">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 16pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
}</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QTextEdit" name="textedit_examdes">
      <property name="styleSheet">
       <string notr="true">QTextEdit{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
	border-radius: 5px;
	border: 1px solid #632d3b;
}</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="button_examremove">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(99, 45, 59);
	font: 13pt &quot;High Tower Text&quot;;
	background: transparent;
	border: 2px solid #632d3b;
	border-radius: 10px;
	padding: 10px 20px;
}
QPushButton:hover{
	background: rgba(255, 255, 255, 0.2)
}
QPushButton:pressed{
	background: rgba(255, 255, 255, 0.1)
}</string>
      </property>
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="button_examedit">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(99, 45, 59);
	font: 13pt &quot;High Tower Text&quot;;
	background: transparent;
	border: 2px solid #632d3b;
	border-radius: 10px;
	padding: 10px 20px;
}
QPushButton:hover{
	background: rgba(255, 255, 255, 0.2)
}
QPushButton:pressed{
	background: rgba(255, 255, 255, 0.1)
}</string>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="button_examadd">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(99, 45, 59);
	font: 13pt &quot;High Tower Text&quot;;
	background: transparent;
	border: 2px solid #632d3b;
	border-radius: 10px;
	padding: 10px 20px;
}
QPushButton:hover{
	background: rgba(255, 255, 255, 0.2)
}
QPushButton:pressed{
	background: rgba(255, 255, 255, 0.1)
}</string>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" rowspan="3">
     <widget class="QListWidget" name="listwidget_examlist">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
	border-radius: 5px;
	border: 1px solid #632d3b;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="lineedit_examname">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
	border-radius: 5px;
	border: 1px solid #632d3b;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="tochoicee">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 25pt &quot;High Tower Text&quot;;
color: #632d3b;
background: transparent;
border: none;
padding: 0px;
}
QPushButton:hover{
background: rgba(255, 255, 255, 0.2);
}
QPushButton:pressed{
background: rgba(255, 255, 255, 0.1);
}
	
</string>
   </property>
   <property name="text">
    <string>â‡½</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
